assessments/unit-01/code.js

```diff
@@ -5,3 +5,42 @@ const emit = (a, b, product) => {
 };
 
 // Write your code here ...
+const averageWeight = (w, n) => (w/n)
+const hypotenuse = (a, b) => (Math.sqrt(a ** 2 + b ** 2))
+const maxRadius = (w, h) => (Math.min (w, h)/2)
+const numCircles = (r,w) => (w / r)
+
+const canSleepIn = (isWeekday, onVacation) => {
+  isWeekday = false
+  onVacation = true
+  return canSleepIn (!isWeekday || onVacation)
```

Comment:
The boolean expression in this line is correct but I'm not sure what that `canSleeepIn` is doing here. You've created an infinite recursion without (probably) even knowing what that is. Just delete it.
```suggestion
  return !isWeekday || onVacation
```